src/main/modules/individualActions.test.js

```diff
@@ -0,0 +1,23 @@
+import { individualActions } from './individualActions';
+
+var Marco = new Object();
+Marco.name = 'Marco';
+Marco.alias = 'PF Squad Soldier';
+Marco.weapon = 'Handgun';
+var PickWeapon = new Object();
+
+test('Individual actions need to be initialized correctly', () => {
+    expect(new individualActions(Marco, PickWeapon)).toEqual({'actionActor': Marco, 'actionType': PickWeapon});
+    expect(new individualActions('Marco', 'PickWeapon', 'Shotgun')).toEqual({'actionActor': 'Marco', 'actionType': 'PickWeapon', 'element': 'Shotgun'});
+});
+
+const testclass = new individualActions;
+
+test('Methods need to be done correctly', () =>{
```

Comment:
The same comment that I left you above, I think it could make the tests easier to understand